src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.db.FruitShopDaoImpl;
+import core.basesyntax.impl.AmountFromFileImpl;
+import core.basesyntax.impl.ConvertingFileToFruitTransactionImpl;
+import core.basesyntax.impl.FruitsFromFileImpl;
+import core.basesyntax.impl.ReadingCsvFileImpl;
+import core.basesyntax.impl.WritingToNewFileImpl;
+import core.basesyntax.service.AmountFromFile;
+import core.basesyntax.service.ConvertingFileToFruitTransaction;
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.FruitsFromFile;
+import core.basesyntax.service.ReadingCsvFile;
+import core.basesyntax.service.WritingToNewFile;
+import java.util.List;
+
+public class Main {
+    // HINT: In the `public static void main(String[] args)`
+    // it is better to create instances of your classes,
+    // and call their methods,
+    // but do not write any business logic in the `main` method!
+    public static void main(String[] args) {
```

Comment:
Create a separate `Main` class with a `main()` method to demonstrate how the program works. Make sure all services are independent and call them in the right order in the `main()` method.